system-parachains/constants/src/kusama.rs

```diff
@@ -23,6 +23,16 @@ pub mod consensus {
 	pub const BLOCK_PROCESSING_VELOCITY: u32 = 1;
 	/// Relay chain slot duration, in milliseconds.
 	pub const RELAY_CHAIN_SLOT_DURATION_MILLIS: u32 = 6000;
+
+	/// Parameters enabling async backing functionality.
+	///
+	/// Once all system chains have migrated to the new async backing mechanism, the parameters
+	/// in this namespace will replace those currently defined in `super::*`.
+	pub mod fast {
+		/// Maximum number of blocks simultaneously accepted by the Runtime, not yet included into
+		/// the relay chain.
+		pub const UNINCLUDED_SEGMENT_CAPACITY: u32 = 3;
```

Comment:
Why is this single constant in `kusama.rs`, instead of bundled with the other constants?

Reply:
I may not understood the comment, but this constant here because it overloads the `super:: UNINCLUDED_SEGMENT_CAPACITY` and stays under `consensus` namespace, later we can drop this module as mentioned above.

Reply:
The question was why this is not in `system-parachains/constants/src/lib.rs`, because it is just a normal parachain config parameter and not really kusama specific.

Reply:
I see, just bc the constant it's overloading is here. I can update it when we enable async backing everywhere.